src/Plug.vala

```diff
@@ -220,6 +226,35 @@ public class About.Plug : Switchboard.Plug {
             critical (e.message);
             hdd = _("Unknown");
         }
+
+        try {
```

Comment:
Ah, we'd talked about pretty-casing these instead of all caps, for more consistency with i.e. .desktop and other keyfiles.